src/app/cli/src/coda.ml

```diff
@@ -255,6 +255,8 @@ let daemon logger =
          ~doc:"full|check|none"
      and plugins = plugin_flag in
      fun () ->
+       (* Immediately disable updating the time offset. *)
+       Block_time.Controller.disable_setting_offset () ;
```

Comment:
It would be a lot safer if it were disabled by default, and you had to enable it on the code paths that we wanted it instead. Is this a lot more annoying? I imagine you only need to do this on specific tests.